dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -227,6 +229,16 @@ COPY debs/sonic-device-data_*.deb debs/libdashapi_*.deb debs/
 
 RUN dpkg -i debs/sonic-device-data_*.deb debs/libdashapi_*.deb
 
+# Install dash-pipeline-utils
+COPY ["files/github_get.py", "/usr/bin"]
+RUN cd /tmp \
+    && python3 /usr/bin/github_get.py https://api.github.com/repos/sonic-net/DASH/contents/dash-pipeline/utils \
```

Comment:
Will we add the commit id here in this PR?

Reply:
Later we may add the commit id if it is necessary regarding the compatibility.